### ISSUE TITLE
Broken list of artists as a result of delete

### ISSUE THREAD

**Issue by ghostx3** [apastel/ytmusic-deleter#30]
Hi Alex, let me say thank you in the very first place for a grate job. Good tool, really. However there is one minor issue which doesn't make me feel 100% good and satisfied of using it.

My original intention was to cleanup my library completely, so I used all options to insure I'm fine and everything is gone, but not everything has gone eventually. 
  - I go a list of artists with broken links on deleted album's songs

![image](https://github.com/apastel/ytmusic-deleter/assets/64723542/646eb289-2d52-4616-9c0d-d6dcd149a828)

If I try to delete the artist I receive

![image](https://github.com/apastel/ytmusic-deleter/assets/64723542/252a2b5f-c1c2-4fcb-b5bd-10e8e2fba917)

and if I try to go to album, of course it does not exists, so I'm getting just a dark screen

![image](https://github.com/apastel/ytmusic-deleter/assets/64723542/76bba8fc-763e-4b76-a5fb-f1c54eb11cbc)

I opened a support request to Goggle, but for multiple weeks they were talking complete crap, like their engineers were not able to reproduce the issue, despite I didn't ask to reproduce, I asked to fix my account ))

If I go ahead and add The Metallica Blacklist again into my library, I see that each artist has 2 songs, but eventually only one I can see.

![image](https://github.com/apastel/ytmusic-deleter/assets/64723542/f69e5500-1932-4512-926c-d69b1ed37b35)

![image](https://github.com/apastel/ytmusic-deleter/assets/64723542/bf8e16f3-c5bf-440f-a2c7-fadf5cb301ff)

I hope a lot that you know their API and could help fix this case using any force method or so, maybe anything else? These broken links are really annoying for me ((

**Comment by apastel** [comment]
Hmm that's odd.
At first I thought maybe the issue is that that track was broken in YT Music to begin with, and it's only after deleting everything else that it got left behind. However in my account I can see that song is there, and I can add The Metallica Blacklist to my library, and I can run the "Remove Library" function in YT Music Deleter and it removes that album successfully for me.

I'm not sure why that song is broken in your library, and unfortunately the tool can only delete songs that the [ytmusicapi](https://github.com/sigma67/ytmusicapi) can find, because this tool is just a wrapper around that API. I don't actually know a lot about Google's API itself; this tool just uses that other project [ytmusicapi](https://github.com/sigma67/ytmusicapi) to do all the interactions with your library, like fetching songs and deleting them. I just tried to make a convenient way to delete everything at once, so sometimes there are weird things that happen that cause songs to get left behind. I've never seen this happen though, where a song cannot be deleted in the browser either.

The only thing I can think of is for you to repost this on the [ytmusicapi](https://github.com/sigma67/ytmusicapi) Github page, and just say that you used the [YTMusic.rate_playlist](https://ytmusicapi.readthedocs.io/en/stable/reference.html#ytmusicapi.YTMusic.rate_playlist) function on each album in your library to delete all your albums, but there is one song leftover from an album that looks like a broken song and cannot be deleted or played. The author of that project might have a better idea of what to do. If they give you a suggestion that involves using their API, I might be able to help you use it.

Good luck, keep me posted how so I can try and help you further. Sorry that's all I got for now.
